speedtest/edu/washington/escience/myria/sqlite/TwitterJoinSpeedTest.java

```diff
@@ -58,7 +59,8 @@ public void before() {
       final Path src = FileSystems.getDefault().getPath(srcPath[i]);
       final Path dst =
           FileSystems.getDefault().getPath(
-              FilenameUtils.concat(getWorkerFolder(workerIDs[i]), "worker_" + workerIDs[i] + "_data.db"));
+              FilenameUtils.concat(DeploymentUtils.getPathToWorkerDir(workingDir, workerIDs[i]), "worker_"
```

Comment:
Another example of how duplicating the worker ID in file names requires unnecessary code.